.github/runner/set-k8s-secrets.sh

```diff
@@ -53,14 +53,14 @@ $2
 }
 
 splitSecretData(){
-  echo "$1" | jq -r 'to_entries | .[] | "\(.key) \(.value)"' | while read K V; do
+  echo "$1" | jq -r 'to_entries | .[] | "\(.key) \(.value)"' | while read -r K V; do
```

Comment:
What's the reason for using `-r`? [Restricted mode](https://www.gnu.org/software/bash/manual/html_node/The-Restricted-Shell.html) prevents "changing directories" and "redirecting output", among other things. Using `read -r` everywhere may make it hard to diagnose why some basic commands don't work.